### PR TITLE
🌠 Add Microsoft Clarity

### PR DESCRIPTION
**Pull request by joshbermanssw**
cc: @landonmaxwell 

As per #2613 wwe have installed microsoft clarity

### CODE REVIEW

File: pages/_document.tsx [hunk 1]
```diff
@@ -44,6 +45,18 @@ export default class MyDocument extends Document {
...
+            dangerouslySetInnerHTML={{
+              __html: `
+                (function(c,l,a,r,i,t,y){
+                  c[a]=c[a]||function(){(c[a].q=c[a].q||[]).push(arguments)};
+                  t=l.createElement(r);t.async=1;t.src="https://www.clarity.ms/tag/"+i;
+                  y=l.getElementsByTagName(r)[0];y.parentNode.insertBefore(t,y);
+                })(window, document, "clarity", "script", "pepjushhm5");
+              `,
+            }}
+          />
```
Comment (isaaclombardssw) [inline pages/_document.tsx:59]: Wonder if we can do it like with Hotjar – https://github.com/tinacms/tina.io/blob/main/pages/_app.tsx (see the useEffect)

**Review by isaaclombardssw — CHANGES_REQUESTED** [review]

File: pages/_document.tsx (hunk 1 above)
Comment (isaaclombardssw) [reply pages/_document.tsx:59]: Also should we remove Hotjar if we're moving to MS clarity.

**Review by isaaclombardssw — APPROVED** [review]
LGTM

File: pages/_document.tsx (hunk 1 above)
Comment (joshbermanssw) [reply pages/_document.tsx]: Done - we're keeping both HotJar and Clarity to compare coverage for now

**Pull request merged**